examples/api-gateway-authorizer/runtime/user.fn.ts

```diff
@@ -0,0 +1,12 @@
+import type { LambdaConfig } from "@notation/aws/lambda.fn";
+import { handle, json } from "@notation/aws/lambda.fn";
+
+export const getUserHandler = handle.apiRequest(() => {
```

Comment:
How can we help the user get the JWT claims from the event? I think the types are optional. Should we have an extra handler like `handle.authorizedApiRequest`  where we made them required?